smoke-tests/src/test/groovy/io/opentelemetry/smoketest/NoopSdkSmokeTest.groovy

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.smoketest
+
+import io.opentelemetry.proto.collector.trace.v1.ExportTraceServiceRequest
+import spock.lang.IgnoreIf
+import spock.lang.Unroll
+
+import java.util.jar.Attributes
+import java.util.jar.JarFile
+
+@IgnoreIf({ os.windows })
+class NoopSdkSmokeTest extends SmokeTest {
+
+  protected String getTargetImage(String jdk) {
+    "ghcr.io/open-telemetry/java-test-containers:smoke-springboot-jdk$jdk-20210218.577304949"
+  }
+
+  @Override
+  protected Map<String, String> getExtraEnv() {
+    return [
+      "otel.javaagent.experimental.sdk.noop": "true"
```

Comment:
Hmm, these are environment variables, so 
```suggestion
      "OTEL_JAVAAGENT_EXPERIMENTAL_SDK_NOOP": "true"
```
would be the preferred naming pattern